### PR TITLE
Make naming less specific to OpenShift

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

**Comment by ashcrow** [comment]
Perused the text changes and they make sense and LGTM.

**Comment by smarterclayton** [comment]
@jwforres can you check my english?

File: pkg/cmd/cli/cli.go [hunk 1]
```diff
@@ -24,9 +24,9 @@ import (
 )
 
 const cliLong = `
-OpenShift Client
+Developer and Administrator Client
 
-The OpenShift client exposes commands for managing your applications, as well as lower level
+The client exposes commands for managing your applications, as well as lower level
```
Comment (jwforres) [inline pkg/cmd/cli/cli.go]: "The client" sounds weird.  Maybe "This client"

File: pkg/cmd/cli/cli.go [hunk 2]
```diff
@@ -159,7 +159,10 @@ func NewCmdKubectl(name string, out io.Writer) *cobra.Command {
 	cmds.Aliases = []string{"kubectl"}
 	cmds.Use = name
 	cmds.Short = "Kubernetes cluster management via kubectl"
-	cmds.Long = cmds.Long + "\n\nThis command is provided for direct management of the Kubernetes cluster OpenShift runs on."
+	cmds.Long = cmds.Long + `
+
+This command exposes the exact semantics of the Kubernetes command line client with support for
```
Comment (jwforres) [inline pkg/cmd/cli/cli.go]: "with additional support for application lifecycle resources."

File: pkg/cmd/cli/cmd/login.go [hunk 3]
```diff
@@ -20,9 +20,9 @@ import (
 
 const (
 	loginLong = `
-Log in to an OpenShift server and save login for subsequent use
+Log in to a Kubernetes server and save login for subsequent use
```
Comment (jwforres) [inline pkg/cmd/cli/cmd/login.go]: is this going to be confusing to end users that dont know openshift is built on kubernetes?

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/login.go]: I'll change it to "Log in to your server ..."

On Thu, Jul 23, 2015 at 3:21 PM, Jessica Forrester <notifications@github.com
> wrote:

> In pkg/cmd/cli/cmd/login.go
> <https://github.com/openshift/origin/pull/3879#discussion_r35360654>:
>
> > @@ -20,9 +20,9 @@ import (
> >
> >  const (
> >  	loginLong = `
> > -Log in to an OpenShift server and save login for subsequent use
> > +Log in to a Kubernetes server and save login for subsequent use
>
> is this going to be confusing to end users that dont know openshift is
> built on kubernetes?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3879/files#r35360654>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift

File: pkg/cmd/cli/cmd/newbuild.go [hunk 4]
```diff
@@ -58,11 +59,11 @@ func NewCmdNewBuild(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
 	}
 
 	cmd.Flags().Var(&config.SourceRepositories, "code", "Source code in the build configuration.")
-	cmd.Flags().VarP(&config.ImageStreams, "image", "i", "Name of an OpenShift image stream to to use as a builder.")
+	cmd.Flags().VarP(&config.ImageStreams, "image", "i", "Name of an image stream to to use as a builder.")
 	cmd.Flags().Var(&config.DockerImages, "docker-image", "Name of a Docker image to use as a builder.")
 	cmd.Flags().StringVar(&config.Name, "name", "", "Set name to use for generated build artifacts")
 	cmd.Flags().StringVar(&config.Strategy, "strategy", "", "Specify the build strategy to use if you don't want to detect (docker|source).")
-	cmd.Flags().BoolVar(&config.OutputDocker, "to-docker", false, "Force the Build output to be DockerImage.")
+	cmd.Flags().BoolVar(&config.OutputDocker, "to-docker", false, "Force the build output to do a generic Docker push.")
```
Comment (jwforres) [inline pkg/cmd/cli/cmd/newbuild.go]: generic?

File: pkg/cmd/experimental/ipfailover/ipfailover.go [hunk 5]
```diff
@@ -16,21 +16,16 @@ import (
...
 the --create flag can be passed to create a deployment configuration that
 will provide IP failover capability. If you are running in production, it is
 recommended that the labeled selector for the nodes matches atleast 2 nodes
 to ensure you have failover protection and that you provide a --replicas=<n>
-value that matches the number of nodes for the given labeled selector.
-
-ALPHA: This command is currently being actively developed. It is intended
-       to simplify the administrative tasks of setting up a highly
-       available failover configuration.`
+value that matches the number of nodes for the given labeled selector.`
```
Comment (jwforres) [inline pkg/cmd/experimental/ipfailover/ipfailover.go]: earlier in the sentence i see "atleast" should be "at least"

Comment (jwforres) [inline pkg/cmd/experimental/ipfailover/ipfailover.go]: to ensure you have have failover protection, and   (add a comma)

File: pkg/cmd/experimental/ipfailover/ipfailover.go [hunk 6]
```diff
@@ -43,7 +38,7 @@ ALPHA: This command is currently being actively developed. It is intended
 
   // Create an IP failover configuration on a selection of nodes labeled
   // "router=us-west-ha" (on 4 nodes with 7 virtual IPs monitoring a service
-  // listening on port 80 (aka the OpenShift router process).
+  // listening on port 80 (such as the router process).
```
Comment (jwforres) [inline pkg/cmd/experimental/ipfailover/ipfailover.go]: think there is a missing closing ) on the line above

File: pkg/cmd/server/admin/create_bootstrappolicy_file.go [hunk 7]
```diff
@@ -36,7 +36,7 @@ func NewCommandCreateBootstrapPolicyFile(commandName string, fullName string, ou
 
 	cmd := &cobra.Command{
 		Use:   commandName,
-		Short: "Create bootstrap policy for OpenShift",
+		Short: "Create default bootstrap policy",
```
Comment (jwforres) [inline pkg/cmd/server/admin/create_bootstrappolicy_file.go]: Create the default bootstrap policy

File: pkg/cmd/server/start/start_allinone.go [hunk 8]
```diff
@@ -178,7 +178,7 @@ func (o AllInOneOptions) Validate(args []string) error {
 	}
 
 	if len(o.MasterArgs.KubeConnectionArgs.ClientConfigLoadingRules.ExplicitPath) != 0 {
-		return errors.New("all-in-one cannot start against with a remote kubernetes, start the master instead")
+		return errors.New("all-in-one cannot start against with a remote Kubernetes server, start the master instead")
```
Comment (jwforres) [inline pkg/cmd/server/start/start_allinone.go]: take out either "against" or "with"

File: pkg/cmd/server/start/start_node.go [hunk 9]
```diff
@@ -260,7 +260,7 @@ func StartNode(nodeConfig configapi.NodeConfig) error {
 	if err != nil {
 		return err
 	}
-	glog.Infof("Starting OpenShift node %s (%s)", config.KubeletServer.HostnameOverride, version.Get().String())
+	glog.Infof("Starting node %s (%s)", config.KubeletServer.HostnameOverride, version.Get().String())
```
Comment (jwforres) [inline pkg/cmd/server/start/start_node.go]: Any reason we say "Starting a node" in one message and "Starting node" in this one

File: pkg/cmd/util/clientcmd/errors.go [hunk 10]
```diff
@@ -14,7 +14,7 @@ const (
 	configurationInvalidReason        = 3
 
 	certificateAuthorityUnknownMsg = "The server uses a certificate signed by unknown authority. You may need to use the --certificate-authority flag to provide the path to a certificate file for the certificate authority, or --insecure-skip-tls-verify to bypass the certificate check and use insecure connections."
-	notConfiguredMsg               = `OpenShift is not configured. You need to run the login command in order to create a default config for your server and credentials:
+	notConfiguredMsg               = `Client is not configured. You need to run the login command in order to create a default config for your server and credentials:
```
Comment (jwforres) [inline pkg/cmd/util/clientcmd/errors.go]: "Your client" or "This client"  sounds weird starting with "Client"

File: pkg/cmd/server/start/start_node.go (hunk 9 above)
Comment (smarterclayton) [inline pkg/cmd/server/start/start_node.go]: Just that in one we have don't identify the node by name and the other we do.

**Comment by smarterclayton** [comment]
[merge]

**Comment by smarterclayton** [comment]
[test]

**Comment by ashcrow** [comment]
Looks like the descriptions in the swagger spec isn't matching:

https://travis-ci.org/openshift/origin/jobs/72404252#L330-L1449

**Comment by smarterclayton** [comment]
For right now I'm going to leave the swagger docs the same because we generate it to the documentation for OpenShift Enterprise.  We need to parameterize it somehow or transform it during documentation.

**Comment by smarterclayton** [comment]
[merge]

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
[merge]

File: docs/generated/oadm_by_example_content.adoc [hunk 11]
```diff
@@ -134,7 +134,7 @@ Install an IP failover group to a set of nodes
 
   // Create an IP failover configuration on a selection of nodes labeled
   // "router=us-west-ha" (on 4 nodes with 7 virtual IPs monitoring a service
-  // listening on port 80 (aka the OpenShift router process).
+  // listening on port 80, such as the router process).
```
Comment (stevekuznetsov) [inline docs/generated/oadm_by_example_content.adoc]: Extra closing paren

**Pull request merged**